README.md

```diff
@@ -78,6 +78,19 @@ Cancel a transaction by id:
 ```typescript
 await arcusClient.cancelTransaction(transactionId)
 ```
+### How to do a release
+
+1. update the changelog
+```
+yarn release
+```
+2. delete the tag created by the release script
+3. create a PR
+4. merge to main with "release and merge" button to avoid merge commit.
```

Comment:
This should be `rebase and merge`, right?

Reply:
xD yes.